openmeetings-web/src/main/java/org/apache/openmeetings/web/common/OmWebSocketPanel.java

```diff
@@ -126,7 +126,7 @@ protected void onMessage(WebSocketRequestHandler handler, TextMessage msg) {
 							}
 								break;
 							case "ping":
-								log.debug("Sending WebSocket PING");
+								log.trace("Sending WebSocket PING");
```

Comment:
I also don't understand how this would be helpful to trace back any particular user without logging any details on who is (or isn't) sending this ping.